### PR TITLE
[YUNIKORN-2194] docs: add `/ws/v1/scheduler/node-utilization` endpoint doc

### PR DESCRIPTION
**Pull request by stu01509**
### What is this PR for?
Add `/ws/v1/scheduler/node-utilization` endpoint doc.

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [x] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/YUNIKORN-2194

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

File: docs/api/scheduler.md [hunk 1]
```diff
@@ -1361,6 +1361,55 @@ Node details include host and rack name, capacity, resources, utilization, and a
...
+        "bucketName": "0-10%",
+        "numOfNodes": 1,
+        "nodeNames": [
+          "aethergpu"
+        ]
+      },
+      {
+        "bucketName": "10-20%",
+        "numOfNodes": 3,
+        "nodeNames": null
```
Comment (wilfred-s) [inline docs/api/scheduler.md:1392]: If we show 3 nodes we should have a list of 3 nodes in the nodeNames also for consistency.

**Review by wilfred-s — CHANGES_REQUESTED** [review]

File: docs/api/scheduler.md (hunk 1 above)
Comment (stu01509) [reply docs/api/scheduler.md]: Hi @wilfred-s 

Thanks for your review, I modified the `numOfNodes` to 2, and some example nodes data.

File: docs/api/scheduler.md [hunk 2]
```diff
@@ -1361,6 +1361,58 @@ Node details include host and rack name, capacity, resources, utilization, and a
...
+## Nodes utilization
+
+Shows how nodes are distributed with regarding the utilization
+
+**URL** : `/ws/v1/scheduler/node-utilization`
+
+**Method** : `GET`
+
+**Auth required** : NO
+
```
Comment (steinsgateted) [inline docs/api/scheduler.md:1373]: missing `### Success response`

**Review by steinsgateted — CHANGES_REQUESTED** [review]

File: docs/api/scheduler.md [hunk 3]
```diff
@@ -1361,6 +1361,58 @@ Node details include host and rack name, capacity, resources, utilization, and a
 }
 ```
 
+## Nodes utilization
```
[the remark this answers is not in the document]
Comment (stu01509) [reply docs/api/scheduler.md]: Ok, Let me adjust the `Nodes` to `Node`.

File: docs/api/scheduler.md [hunk 4]
```diff
@@ -1361,6 +1361,60 @@ Node details include host and rack name, capacity, resources, utilization, and a
 }
 ```
 
+## Node utilization
+
+Shows how node are distributed with regarding the utilization
```
Comment (steinsgateted) [inline docs/api/scheduler.md:1366]: There are grammatical errors, maybe change it to this:
`Show how every node is distributed with regard to utilization.`

**Comment by steinsgateted** [comment]
LGTM

**Review by steinsgateted — APPROVED** [review]

File: docs/api/scheduler.md [hunk 5]
```diff
@@ -1361,6 +1361,60 @@ Node details include host and rack name, capacity, resources, utilization, and a
 }
 ```
 
+## Node utilization
+
+Show how every node is distributed with regard to utilization.
+
```
Comment (chenyulin0719) [inline docs/api/scheduler.md:1367]: Sorry for late review, but I think we should emphasize the utilization type is 'dominant' resource type only.

Some thing like: 
"Show how every node is distributed with regard to dominant resource utilization."
(Dominant resource is defined as the highest utilised resource type on the root queue based on the registered resources.)

https://github.com/apache/yunikorn-core/blob/master/pkg/webservice/handlers.go#L353-L356

Comment (stu01509) [reply docs/api/scheduler.md]: Thanks for your review, updated the commit.

**Review by chenyulin0719 — APPROVED** [review]
LGTM

File: docs/api/scheduler.md [hunk 6]
```diff
@@ -1361,6 +1361,60 @@ Node details include host and rack name, capacity, resources, utilization, and a
...
+**Auth required** : NO
+
+### Success response
+
+**Code** : `200 OK`
+
+**Content examples**
+
+```text
```
Comment (chia7712) [inline docs/api/scheduler.md:1380]: it should be json

**Review by chia7712 — COMMENTED** [review]
@stu01509 nice patch. Will merge it if one last commit gets addressed

File: docs/api/scheduler.md (hunk 6 above)
Comment (chenyulin0719) [reply docs/api/scheduler.md:1380]: Hi @chia7712 , I will fix in [YUNIKORN-2316](https://issues.apache.org/jira/browse/YUNIKORN-2316).

Comment (chia7712) [reply docs/api/scheduler.md:1380]: merge into master, @chenyulin0719 please fix this trivial comment in the follow-up.

**Pull request closed without merging**